### DISCUSSION TITLE
blog/2021/learning-to-hugo-content-organisation/

### DISCUSSION THREAD

**Discussion by justinjbird** [justinjbird/site-blog-comments#38] [General]
# blog/2021/learning-to-hugo-content-organisation/

Directory structure and content organisation

https://www.justinjbird.me/2021/learning-to-hugo-content-organisation/

**Comment by markherring** [comment]
Hey Justin great blog.. I am new to hugo and enjoyed this section. I am still not sure how to be able to do:
URL/blog (get all blog content)
URL/blog/2022 (get blog content from just 2022) etc. 
Would like this to be automated, ie no html file in each subdirectory. My directory structure is similar to yours.
Let me know if you have done this.

**Comment by justinjbird** [comment]
Hey @markherring! Sorry I didn't notice this comment before. I've just implemented comments on my revised site using giscus and spotted this whilst I was migrating! Did you get this working?
